assets/i18n/qqq.json

```diff
@@ -500,7 +500,8 @@
 	"I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_2": "Text on the Teach page outlining Oppia's Publication Policy.",
 	"I18N_TEACH_PAGE_TAB_PARTICIPATION_PLAYBOOK": "Text on the tab for Participation Playbook on the Teach page.",
 	"I18N_TEACH_PAGE_TAB_TEACH_WITH_OPPIA": "Text on the tab for Teach with Oppia on the Teach page.",
-	"I18N_TOPNAV_ABOUT": "Text displayed in the navigation bar. - When the user hovers over the button, a list with several information pages is displayed.\n{{Identical|About}}",
+	"I18N_TOPIC_VIEWER_PRACTICE": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the subtopics that lie within the topic.\n{{Identical|Practice}}",
+	"I18N_TOPIC_VIEWER_STORY": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the stories that lie within the topic.\n{{Identical|Story}}","I18N_TOPNAV_ABOUT": "Text displayed in the navigation bar. - When the user hovers over the button, a list with several information pages is displayed.\n{{Identical|About}}",
```

Comment:
Move I18N_TOPNAV_ABOUT to the next line.

Reply:
Done.